stigma/src/main/java/io/nixer/nixerplugin/stigma/login/StigmaLoginActivityHandler.java

```diff
@@ -43,6 +42,9 @@ public StigmaLoginActivityHandler(final HttpServletRequest request,
     @Override
     public void handle(final LoginContext context) {
         Assert.notNull(context, "LoginContext can not be null");
+        if (context.getLoginResult() == null) {
```

Comment:
Why not `Assert.notNull(context.getLoginResult(), ...`? ;)

Reply:
because I wanted IllegalStateException - LoginContext should always have a LoginResult. Though IllegalArgumentException would also be fine.